guide/gnucobpg.texi

```diff
@@ -90,9 +90,9 @@ Permission is granted to copy and distribute translations of this manual into an
 @comment *********************************************************************
 @comment ** TOP NODE POINT (Ignore if running TeX)                          **
 @comment *********************************************************************
-@ifnottex
-@validatemenus off
-@end ifnottex
+@comment @ifnottex
+@comment @validatemenus off
+@comment @end ifnottex
```

Comment:
this can be dropped, no need for a comment

Reply:
done